config/webhook/manifests.yaml

```diff
@@ -40,6 +40,7 @@ webhooks:
     apiVersions:
     - v1
     operations:
+    - CREATE
```

Comment:
With failurePolicy: Fail, if the webhook service isn't running at initial deployment, this will prevent enabling events?

Reply:
By default we don't enable events. 15 secs should be good enuff  to check for its availability

Reply:
once default config is created only then user can modify or apply changes to CR to enable events